src/TropicalGeometry/positive_variety.jl

```diff
@@ -0,0 +1,73 @@
+@doc raw"""
+    positive_tropical_variety(I::MPolyIdeal,nu::TropicalSemiringMap)
+
+Return the positive tropical variety of `I` as a `PolyhedralComplex` as per the definition in [SW05](@cite).  Assumes that `I` is generated either by binomials or by linear polynomials and that `I` is defined either over
+(a) the rational numbers and that `nu` encodes the trivial valuation,
+(b) the rational function field over the rational numbers and that `nu` encodes the t-adic valuation.
```

Comment:
Have you checked by locally building the docs that this produces what you expect? From the first glance, I think that this may do weird linebreaks. Maybe one of the following is suitable for your case here?
```suggestion
1. the rational numbers and that `nu` encodes the trivial valuation,
2. the rational function field over the rational numbers and that `nu` encodes the t-adic valuation.
```
```suggestion
- the rational numbers and that `nu` encodes the trivial valuation,
- the rational function field over the rational numbers and that `nu` encodes the t-adic valuation.
```